src/xmm_utils/multiprocessing.py

```diff
@@ -19,10 +19,11 @@ def mp_run(func, kwds_generator: Generator, num_processes: int, debug: bool = Fa
         pool.join()
     end = datetime.now()
     results = []
-    flattened_apply_results = [
-        item for sublist in apply_results for item in (sublist if isinstance(sublist, list) else [sublist])
-    ]
-    for apply_result in flattened_apply_results:
-        if apply_result is not None:
-            results.append(apply_result)
```

Comment:
I changed this because the multiprocessing crashed in the previous version, which seems to be the same as this change. It tried to run `.get()` on lists. Have you tested this on every stage of the pipeline in the docker?

Reply:
I did. Everything worked fine for me. I‘ve used the previous version to create the whole dataset without any errors occurring.

Reply:
Just ran it again. Everything works fine for me.